### ISSUE TITLE
Improving https://ocaml.org/docs/index.fr.html

### ISSUE THREAD

**Issue by gurleennsidhuu** [ocaml/v2.ocaml.org#1409]
https://ocaml.org/docs/index.fr.html page looks a little unorganized here:
![Screenshot 2021-04-05 110037](https://user-images.githubusercontent.com/64313783/113540719-5aa61e00-95fe-11eb-98f7-b0bcc3d41be0.png)
I would like to fix it to:
![Screenshot 2021-04-05 110328](https://user-images.githubusercontent.com/64313783/113540817-917c3400-95fe-11eb-879a-af2c4308ec5e.png)

**Comment by pitag-ha** [comment]
Good catch! I'd say you can start working on it, if you'd like :) In fact, the change you've proposed is how the English site looks like.
